hbsir/metadata.py

```diff
@@ -96,6 +107,23 @@ class Metadatas:
     schema = open_yaml("metadata/schema.yaml")
     other = open_yaml("metadata/other.yaml")
 
+    def reload_schema(self):
+        self.schema = open_yaml("metadata/schema.yaml")
+
+
+load_table_defaults = settings["functions_defaults"]["load_table"]
+
+
+class LoadTable(BaseModel):
```

Comment:
Shouldn't this be dataclass?

Reply:
It seems that it is possible to use dataclass in Pydantic, as mentioned in their documentation: https://docs.pydantic.dev/latest/usage/dataclasses/.
But it is probably better to consider this later.